front/app/components/ParticipationCTABars/ParticipationCTAContent/index.tsx

```diff
@@ -48,14 +48,21 @@ type Props = {
   hasUserParticipated?: boolean;
   CTAButton?: React.ReactNode;
   currentPhase: IPhaseData | undefined;
+  hideDefaultParticipationMessage?: boolean;
+  participationState?: JSX.Element; // Optional element which displays on bottom left
```

Comment:
Open to renaming this - essentially it's a spot in the bar for the "live view" of the ongoing participation :thinking: